requestmanager/asyncloader/responsecache/responsecache.go

```diff
@@ -43,14 +43,20 @@ func New(unverifiedBlockStore UnverifiedBlockStore) *ResponseCache {
 
 // FinishRequest indicate there is no more need to track blocks tied to this
 // response
-func (rc *ResponseCache) FinishRequest(requestID graphsync.RequestID) {
+func (rc *ResponseCache) FinishRequest(requestID graphsync.RequestID) uint64 {
```

Comment:
add comment indicating this return value is the total length of data over the blocks that have been pruned? (if i'm reading that right)